### PR TITLE
Modify ECSN

### PR DESCRIPTION
**Pull request by XinlongSBU**

### CODE REVIEW

File: Exec/SCIENCE/ecsn/GNUmakefile [hunk 1]
```diff
@@ -20,7 +20,7 @@ MICROPHYSICS_HOME ?= ../../../../Microphysics
 # If not we use the version in $(MICROPHYSICS_HOME)/Microphysics/(EOS/conductivity/networks)
 EOS_DIR          := helmholtz
 CONDUCTIVITY_DIR := stellar
-NETWORK_DIR      := ECSN
+NETWORK_DIR      := ECSN_Donald
```
Comment (zingale) [inline Exec/SCIENCE/ecsn/GNUmakefile]: is this really the new network name?

File: Exec/SCIENCE/ecsn/input_ecsn [hunk 2]
```diff
@@ -1,10 +1,10 @@
 # INITIAL MODEL
-maestro.model_file = "kepler_xinlong_ONe6040-final.hse.5120"
-maestro.drdxfac = 5
+maestro.model_file = "ECSN-ONe6040-final.hse.20480"
+maestro.drdxfac = 80
```
Comment (zingale) [inline Exec/SCIENCE/ecsn/input_ecsn]: we really don't recommend drdxfac that high.  You should use the normal factor of 5.  There won't be enough information to get a meaningful average when you use 80.

File: Exec/SCIENCE/ecsn/input_ecsn [hunk 3]
```diff
@@ -51,7 +56,7 @@ maestro.use_divu_firstdt = true
 # 0 = Interior   3 = Symmetry
 # 1 = Inflow     4 = Slipwall
 # 2 = Outflow    5 = NoSlipWall
-maestro.lo_bc = 3 3 3
+maestro.lo_bc = 2 2 2
 maestro.hi_bc = 2 2 2
```
Comment (zingale) [inline Exec/SCIENCE/ecsn/input_ecsn]: for an octant, you need to have reflecting boundaries on the lower faces, to enforce the symmetry.

File: Exec/SCIENCE/ecsn/GNUmakefile (hunk 1 above)
Comment (XinlongSBU) [reply Exec/SCIENCE/ecsn/GNUmakefile]: Yes, ECSN_Donald is the newest network. I can modify the name to ECSN. and delete the old version.

File: Exec/SCIENCE/ecsn/input_ecsn (hunk 2 above)
Comment (XinlongSBU) [reply Exec/SCIENCE/ecsn/input_ecsn]: I see. I will do 'maestro.drdxfac = 5'.

File: Exec/SCIENCE/ecsn/input_ecsn (hunk 3 above)
Comment (XinlongSBU) [reply Exec/SCIENCE/ecsn/input_ecsn]: I am not using octant now. I am using 512 resolution and tagging.f90. So I set "maestro.lo_bc" back to 2 2 2.

Comment (zingale) [reply Exec/SCIENCE/ecsn/input_ecsn]: oh sorry, I didn't see that that was commented out.

File: Exec/SCIENCE/ecsn/GNUmakefile (hunk 1 above)
Comment (zingale) [reply Exec/SCIENCE/ecsn/GNUmakefile]: yes, please just update ECSN without renaming.

Comment (dwillcox) [reply Exec/SCIENCE/ecsn/GNUmakefile]: What was the difference between ECSN and ECSN_Donald?

Comment (XinlongSBU) [reply Exec/SCIENCE/ecsn/GNUmakefile]: I just check. In my local machine, ECSN_Donald is modified by you but ECSN is not. But they are the same in github.

Comment (dwillcox) [reply Exec/SCIENCE/ecsn/GNUmakefile]: Oh okay, that was probably back when we had an old and new version of the ECSN network before and after the changes to the energy generation. But now those changes are in the development branch of the Microphysics repository for the ECSN network.

File: Exec/SCIENCE/ecsn/input_ecsn (hunk 2 above)
Comment (XinlongSBU) [reply Exec/SCIENCE/ecsn/input_ecsn]: When I use 'maestro.drdxfac = 5', this two below will not be equal. Is this fine?

model file mapping (spherical base state)
 dr of MAESTRO base state =                                 78125.00000
 dr of input file data =                                    9765.625000
@dwillcox @zingale

Comment (zingale) [reply Exec/SCIENCE/ecsn/input_ecsn]: no.  You need to regenerate the initial model at the base state resolution.

Basically, drdxfac = 5 means that the base state resolution will be 5x finer than whatever the finest level's resolution (dx) is.

Comment (XinlongSBU) [reply Exec/SCIENCE/ecsn/input_ecsn]: Yeah, I talked to Don and understand.now.

**Pull request closed without merging**